test/windows/static.test.ts

```diff
@@ -0,0 +1,269 @@
+import * as path from "path";
+import { test } from "tap";
+
+import * as plugin from "../../lib";
+import { DepTree, ImageType, PluginResponseStatic } from "../../lib/types";
+
+const getFixture = (fixturePath) =>
+  path.join(__dirname, "../fixtures/docker-archives", fixturePath);
+
+test("static analysis builds the expected response", async (t) => {
+  const thisIsJustAnImageIdentifierInStaticAnalysis = "node:doesnotexist";
+  const dockerfile = undefined;
+  const pluginOptionsWithSkopeoCopy = {
+    staticAnalysisOptions: {
```

Comment:
I think all the Windows test cases with `staticAnalysisOptions` are not needed: this is a path used only by `kubernetes-monitor` which won't ever run on Windows, so I think we don't need them 🙏